t/t0005-rexec.t

```diff
@@ -0,0 +1,190 @@
+#!/bin/sh
+#
+
+test_description='Test broker rexec functionality
+
+
+Test rexec functionality
+'
+
+. `dirname $0`/sharness.sh
+SIZE=4
+test_under_flux ${SIZE} minimal
+
+TEST_SUBPROCESS_DIR=${FLUX_BUILD_DIR}/src/common/subprocess
+
+test_expect_success 'basic rexec functionality (process success)' '
+	${FLUX_BUILD_DIR}/t/rexec/rexec /bin/true
+'
+
+test_expect_success 'basic rexec functionality (process fail)' '
+	! ${FLUX_BUILD_DIR}/t/rexec/rexec /bin/false
```

Comment:
Use of `test_must_fail` is preferred to `!`, from `sharness.sh`:
>  Use it instead of "! <command>". For example, when <command> dies due to a
>  segfault, test_must_fail diagnoses it as an error, while "! <command>" would
>  mistakenly be treated as just another expected failure.

However, I'm not pedantic on this point, just wanted to point it out for future reference...